doc/tutorials/introduction/crosscompilation/crosscompile_with_multiarch.markdown

```diff
@@ -0,0 +1,573 @@
+--     Version control (extra):     faa5468
+--
+--   Platform:
+--     Timestamp:                   2023-12-01T22:02:14Z
+--     Host:                        Linux 6.5.0-13-generic x86_64
+--     Target:                      Linux 1 aarch64
+--     CMake:                       3.27.4
+--     CMake generator:             Ninja
+--     CMake build tool:            /usr/bin/ninja
+--     Configuration:               Release
+--
+--   CPU/HW features:
+--     Baseline:                    NEON FP16
+--       required:                  NEON
+--       disabled:                  VFPV3
+--     Dispatched code generation:  NEON_DOTPROD NEON_FP16 NEON_BF16
+--       requested:                 NEON_FP16 NEON_BF16 NEON_DOTPROD
+--       NEON_DOTPROD (1 files):    + NEON_DOTPROD
+--       NEON_FP16 (2 files):       + NEON_FP16
+--       NEON_BF16 (0 files):       + NEON_BF16
+--
+--   C/C++:
+--     Built as dynamic libs?:      YES
+--     C++ standard:                11
+--     C++ Compiler:                /usr/bin/aarch64-linux-gnu-g++  (ver 13.2.0)
+
+:
+:
+
+--
+--   Video I/O:
+--     DC1394:                      NO
+--     FFMPEG:                      YES
+--       avcodec:                   YES (60.3.100)
+--       avformat:                  YES (60.3.100)
+--       avutil:                    YES (58.2.100)
+--       swscale:                   YES (7.1.100)
+--       avresample:                NO
+--     GStreamer:                   NO
+--     v4l/v4l2:                    YES (linux/videodev2.h)
+--
+@endcode
+
+If enabling python3 wrapper is succeeded, `Pyhton 3:` section shows more.
+
+@code{.unparsed}
+--
+--   Python 3:
+--     Interpreter:                 /usr/bin/python3.11 (ver 3.11.6)
+--     Libraries:                   /usr/lib/aarch64-linux-gnu/libpython3.11.so
+--     numpy:                       /usr/local/lib/python3.11/dist-packages/numpy/core/include/ (ver undefined - cannot be probed because of the cross-compilation)
+--     install path:                lib/python3.11/dist-packages/cv2/python-3.11
+--
+--   Python (for build):            /usr/bin/python3.11
+--
+@endcode
+
+### Step3) Build and archive OpenCV libraries and headers
+This step in in host.
+
+Build and install.
+(This `install` means only that copying artifacts to `install` folder.)
+
+@code{.bash}
+     cmake --build   build4-full_arm64
+sudo cmake --install build4-full_arm64
+@endcode
+
+Archive artifacts(built libraries and headers) to `opencv_arm64.tgz` with tar command.
+
+@code{.bash}
+tar czvf opencv_arm64.tgz -C build4-full_arm64/install .
+@endcode
+
+And send `opencv_arm64.tgz` to target.
+
+### Step4) Install dependency libraries at target
+This step is on target.
+
+Install dependency libraries for OpenCV/OpenCV contrib libraies at target.
+
+@code{.bash}
+sudo apt install -y \
+    libavcodec-dev \
+    libavformat-dev \
+    libavutil-dev \
+    libswscale-dev \
+    libfreetype-dev \
+    libharfbuzz-dev
+
+sudo ldconfig
+@endcode
```

Comment:
Technically, you do not need -dev packages on the target system. binary part is enough.